docs/stats.md

```diff
@@ -0,0 +1,38 @@
+# Stats
+
+## Overview
+<!-- TODO: move module definitions here? -->
+A set of methods used to calculate statistics over the course of an
+experiment.
+Each such method is a higher-order function returning a `track_stat` method
+that takes in an Experiment object, an action key, and the address of the
+Keeper that executed said action.
+The higher-order function can take in whatever parameters it may need to
+provide to `track_stat`.
+Each `track_stat` function will be run after a Keeper executes in an action
+in a timestep, so an intended design pattern is to guard your logic in
+`track_stat` w/ a predicate that ensures you're only executing the method
+in response to a specific action you were listening for.
```

Comment:
yup nice, also prob worth mentioning that all the `track_stat` methods will get run at the very beginning, and very end, of each timestep - in case there are e.g. stats you need to reset at the beginning of each timestep or those which you need to measure only at the end

Reply:
I say this in `experiment.md` in lines 27-31 but let me know if I should make it more clear, or add it to here.